app/views/pages/public.html.erb

```diff
@@ -0,0 +1,6 @@
+<section class="jumbotron text-center">
+  <div class="container">
+    <h1 class="jumbotron-heading">Taste of Code</h1>
+    <p class="lead text-muted">Beginner coding workshop</p>
+  </div>
+</section>
```

Comment:
I still don't see how you are rendering this page.
I just checked your routes and there is no `/public` URL in there.

Will you be adding that in another pull?